src/app/cli/src/graphql.ml

```diff
@@ -444,13 +446,13 @@ module Types = struct
                 @@ Option.value_exn account.Account.Poly.public_key )
           ; field "balance"
               ~typ:(non_null AnnotatedBalance.obj)
-              ~doc:"A balance of Coda as a stringified uint64"
+              ~doc:"The number of coda tokens owned by the account"
               ~args:Arg.[]
               ~resolve:(fun _ {account; _} -> account.Account.Poly.balance)
           ; field "nonce" ~typ:string
               ~doc:
-                "Nonces are natural numbers that increase each transaction. \
-                 Stringified uint32"
+                "A natural number that increases with each transaction \
```

Comment:
To be more precise, should it be:

`A natural number that increases with each transaction signed by account owner`?

Reply:
I feel like we can include more details in a wiki entry or something